src/applications/public-outreach-materials/libraries/library-filters.js

```diff
@@ -55,9 +70,47 @@ export function libraryCurrent() {
   });
 }
 
+export function libraryPagerGen() {
+  numCards = libraryNumCards();
+
+  if (document.getElementById('pager-nums-insert')) {
+    const diff = pages - activePage;
+    let pagerHtml;
+
+    pagerHtml = `<button id="va-pagination-active-num" class="va-page-numbers">
+    ${activePage}</button>`;
+    if (diff > 1 && (numCards === undefined || numCards > 9)) {
+      pagerHtml += `<a class="pager-numbers" aria-label="Load page
+      ${activePage + 1}" tabindex="${activePage + 1}">${activePage + 1}</a>`;
+    }
+    if (diff > 2 && (numCards === undefined || numCards > 9)) {
+      pagerHtml += `<a class="pager-numbers"" aria-label="Load page
```

Comment:
Just noticed this string and the one below have an extra `"` after `class="pager-numbers"`. Perfect example of what I was thinking of in the comment I just made for pulling this out into a function! :smile:

Reply:
Good catch - removing those doubles, @cvalarida :+1: